driver/bpf/fillers.h

```diff
@@ -1035,7 +1035,48 @@ FILLER(sys_getrlimit_setrlimit_e, true)
 	return bpf_push_u8_to_ring(data, rlimit_resource_to_scap(resource));
 }
 
-FILLER(sys_getrlimit_setrlrimit_x, true)
+FILLER(sys_getrlimit_x, true)
+{
+	unsigned long val;
+	long retval;
+	s64 cur;
+	s64 max;
+	int res;
+
+	/* Parameter 1: ret (type: PT_ERRNO) */
+	retval = bpf_syscall_get_retval(data->ctx);
+	res = bpf_push_s64_to_ring(data, retval);
+	CHECK_RES(res);
+
+	/*
+	 * Copy the user structure and extract cur and max
+	 */
+	if(retval >= 0 || data->state->tail_ctx.evt_type == PPME_SYSCALL_SETRLIMIT_X)
+	{
+		struct rlimit rl;
+
+		val = bpf_syscall_get_argument(data, 1);
+		if(bpf_probe_read_user(&rl, sizeof(rl), (void *)val))
+			return PPM_FAILURE_INVALID_USER_MEMORY;
```

Comment:
Instead of returning this code probably here we want to return `cur = 0;` and `max = 0;` See the modern bpf implementation. Same for the kmod

Reply:
We can do something like this 
```c
		struct rlimit rl = {0};
		val = bpf_syscall_get_argument(data, 1);
		bpf_probe_read_user(&rl, sizeof(rl), (void *)val);
		cur = rl.rlim_cur;
		max = rl.rlim_max;
```

instead of returning `PPM_FAILURE_INVALID_USER_MEMORY` if we fail in `bpf_probe_read_user`